src/components/RegisterForm.tsx

```diff
@@ -28,32 +32,18 @@ function RegisterForm(props: RegisterFormProps) {
     actions.setSubmitting(false);
     setLoading(true);
     try {
-      const data = {
+      const data: Attendee = {
+        eventID: id,
         ...values,
       };
-
-      //await registerEvent(data,id);
-      toast.success("You have registered successfully!", {
-        position: "top-right",
-        autoClose: 5000,
-        hideProgressBar: false,
-        closeOnClick: true,
-        pauseOnHover: true,
-        draggable: true,
-        theme: "colored",
-      });
-    } catch {
-      toast.error(
-        "There was an error with your registration, please try again later",
-        {
-          position: "top-right",
-          autoClose: 5000,
-          hideProgressBar: false,
-          closeOnClick: true,
-          pauseOnHover: true,
-          draggable: true,
-          theme: "colored",
-        }
+      console.log(data);
```

Comment:
Could you use a known logging package here instead of using a bare-bone `console.log` statement With a logging package you can customize how you logging would be.

Take a look at [this package](https://www.npmjs.com/package/winston).

Reply:
<img width="856" alt="image" src="https://github.com/user-attachments/assets/0e5da8bc-2567-41fc-8fb2-8227eee70eff">


the logger can't be used on the client side , as it throws an "Getting "Module not found: Can't resolve 'fs' " error.

it can be used only on the server side or if i create api endpoints to access the logger, should i do that?